tools/experiments/experiments-config.js

```diff
@@ -53,6 +53,11 @@ export const EXPERIMENTS = [
     name: 'AMP extension for a basic, flexible, carousel',
     spec: 'https://github.com/ampproject/amphtml/issues/20595',
   },
+  {
+    id: 'amp-lightbox',
```

Comment:
Please skip this file for now. We'll do a pass over the experiment names/groupping later for all bento components.